Libplanet/Net/Messages/Message.cs

```diff
@@ -11,6 +11,19 @@ namespace Libplanet.Net.Messages
 {
     internal abstract class Message
     {
+        internal const int HeaderSize = 4;
+
+        protected const bool LeaveTrail = true;
```

Comment:
I guess it's okay that we omit this constant and use `true` or `false` directly. if we want to show its meaning, `base(trail: true)` is enough.